BE/app/domain/auth/auth_service.py

```diff
@@ -1,11 +1,9 @@
 from typing import Optional
```

Comment:
`auth` is a generic concept. it supports the protocol layer only. it shouldnt be DOMAIN